cinema/tests/test_movie_api.py

```diff
@@ -43,9 +43,7 @@ def sample_actor(**params):
 
 
 def sample_movie_session(**params):
-    cinema_hall = CinemaHall.objects.create(
-        name="Blue", rows=20, seats_in_row=20
-    )
+    cinema_hall = CinemaHall.objects.create(name="Blue", rows=20, seats_in_row=20)
```

Comment:
never change tests, when you didn't write them and wasn't explicitly asked for that. Even if it's only styling.